roles/controller_casc_from_aap/tasks/all.yml

```diff
@@ -52,11 +52,14 @@
     - name: "Export Workflow Job Templates"
       include_tasks: "workflow_job_templates_from_aap_improved.yml"
       when: "'workflow_job_templates' in input_tag or 'all' in input_tag"
+    - name: "Export Projects"
+      include_tasks: "projects_from_aap _improved.yml"
```

Comment:
There is a blank space between app and _improved

Reply:
Fixed

Reply:
Fixed also blank space on the file